rust/src/smb/detect.rs

```diff
@@ -1,4 +1,4 @@
-/* Copyright (C) 2017 Open Information Security Foundation
+/* Copyright (C) 2023 Open Information Security Foundation
```

Comment:
Copyright years are updated in the form of: initial year-last updated year
See https://github.com/OISF/suricata/pull/9415/commits/482eb11213677d4f2b56e52757b5810b112cd67d#diff-809d44481dd655fd192c9767d520c5ced78d5c1de6f2b94ee4255e91c1376d53L1 ;)

Reply:
updated in next PR